frodo.js

```diff
@@ -1,11 +1,11 @@
-const uniqueSortedArray = [...new Set(array)].sort();
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?